kubernetes-services/work-scheduler/app/routers/router.ts

```diff
@@ -0,0 +1,19 @@
+import express, { NextFunction } from 'express';
+
+/**
+ * Set up routes for the service. These are used exclusively by Kubernetes.
+ * @returns Router configured with service routes.
+ */
+export default function router(): express.Router {
+  const result = express.Router();
+
+  result.get('/liveness', async (req, res, _next: NextFunction): Promise<void> => {
```

Comment:
nit - only documenting the type of `_next`, but not `req` or `res` - I'm sure this is copied off of something else though.

Reply:
Fixed